modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2507,16 +2507,16 @@ def _copartition(self, axis, other, how, sort, force_repartition=False):
         return (reindexed_frames[0], reindexed_frames[1:], joined_index, base_lengths)
 
     @lazy_metadata_decorator(apply_axis="both")
-    def binary_op(self, op, right_frame, join_type="outer"):
+    def nary_op(self, op, right_frames: list, join_type="outer"):
```

Comment:
What does this name mean?

Reply:
should we name it `n_ary_op` to signify that naming?..

Reply:
Yep, done.